includes/utils.inc

```diff
@@ -1055,8 +1055,9 @@ function wf_crm_get_fields($var = 'fields') {
     if (isset($sets['membership'])) {
       $fields['membership_membership_type_id'] = array(
         'name' => t('Membership Type'),
-        'type' => 'select',
+        'type' => 'civicrm_select',
         'expose_list' => TRUE,
+        'civicrm_live_options' => 1,
         'extra' => array('civicrm_live_options' => 1),
```

Comment:
Hm, maybe this is the source of the weird key in the Yaml?

Reply:
Like comment below, the `extra` is legacy and can be removed (yay!)